src/ansys/aedt/core/maxwell.py

```
@@ -2135,6 +2135,176 @@ def create_setup(self, name="MySetupAuto", setup_type=None, **kwargs):
         setup.update()
         return setup
 
+    @pyaedt_function_handler(file_path="output_file", setup_name="setup")
+    def export_rl_matrix(
+        self,
+        matrix_name,
+        output_file,
+        is_format_default=True,
+        width=8,
+        precision=2,
+        is_exponential=False,
+        setup=None,
+        default_adaptive="LastAdaptive",
+        is_post_processed=False,
+    ):
+        """Export R/L matrix after solving.
+
+        Parameters
+        ----------
+        matrix_name : str
+            Matrix name to be exported.
+        output_file : str
+            Output file path to export R/L matrix file to.
+            Extension must be ``.txt``.
+        is_format_default : bool, optional
+            Whether the exported format is default or not.
+            If False the custom format is set (no exponential).
+        width : int, optional
+            Column width in exported .txt file.
+        precision : int, optional
+            Decimal precision number in exported \\*.txt file.
+        is_exponential : bool, optional
+            Whether the format number is exponential or not.
+        setup : str, optional
+            Name of the setup.
+            If not provided, the active setup is used.
+        default_adaptive : str, optional
+            Adaptive type.
+            The default is ``"LastAdaptive"``.
+        is_post_processed : bool, optional
+            Boolean to check if it is post processed. Default value is ``False``.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+        """
+        if self.solution_type not in [
+            SOLUTIONS.Maxwell2d.EddyCurrentXY,
+            SOLUTIONS.Maxwell2d.EddyCurrentZ,
+            SOLUTIONS.Maxwell3d.EddyCurrent,
+        ]:
+            self.logger.error("RL Matrix can only be exported if solution type is Eddy Current.")
```

Could you raise an exception here ?